Source/WebGPU/WebGPU/Instance.h

```diff
@@ -34,6 +34,8 @@
 #import <wtf/TZoneMalloc.h>
 #import <wtf/ThreadSafeRefCounted.h>
 #import <wtf/WeakPtr.h>
+#include "WebGPU.h"
+#include "WebGPUExt.h"
```

Comment:
not strictly necessary, but generally WebGPU.h / WebGPUExt.h is imported like:

```
#import <WebGPU/WebGPU.h>
#import <WebGPU/WebGPUExt.h> 
```
but perhaps with continued changes to these headers, we should stop doing that.

Probably changing `include` -> `import` would be reasonable